From 7c353a43fd4024e76b989d3d5c15ce7b70800b95 Mon Sep 17 00:00:00 2001
From: Jukka Honkela <jukka.honkela@palm.com>
Date: Thu, 4 Oct 2012 14:13:25 +0300
Subject: [PATCH] Upstart 1.5 compatibility changes

Open-webOS-DCO-1.0-Signed-off-by: Jukka Honkela <jukka.honkela@palm.com>
---
 files/launch/activitymanager.conf.in |   25 +++++++++++++++++++++++++
 files/launch/activitymanager.in      |   25 -------------------------
 2 files changed, 25 insertions(+), 25 deletions(-)
 create mode 100644 files/launch/activitymanager.conf.in
 delete mode 100644 files/launch/activitymanager.in

diff --git a/files/launch/activitymanager.conf.in b/files/launch/activitymanager.conf.in
new file mode 100644
index 0000000..64a26e7
--- /dev/null
+++ b/files/launch/activitymanager.conf.in
@@ -0,0 +1,25 @@
+# -*- mode: conf; -*-
+# @@@LICENSE
+#
+#      Copyright (c) 2009-2012 Hewlett-Packard Development Company, L.P.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# LICENSE@@@
+
+start on datastore-initialized
+stop on started start_update
+
+respawn
+exec @WEBOS_INSTALL_SBINDIR@/activitymanager -c {\"log\":{\"appender\":{\"type\":\"syslog\"},\"levels\":{\"default\":\"warning\"}}}
+
diff --git a/files/launch/activitymanager.in b/files/launch/activitymanager.in
deleted file mode 100644
index 64a26e7..0000000
--- a/files/launch/activitymanager.in
+++ /dev/null
@@ -1,25 +0,0 @@
-# -*- mode: conf; -*-
-# @@@LICENSE
-#
-#      Copyright (c) 2009-2012 Hewlett-Packard Development Company, L.P.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# LICENSE@@@
-
-start on datastore-initialized
-stop on started start_update
-
-respawn
-exec @WEBOS_INSTALL_SBINDIR@/activitymanager -c {\"log\":{\"appender\":{\"type\":\"syslog\"},\"levels\":{\"default\":\"warning\"}}}
-
-- 
1.7.9.5

